packages/common/src/hardforks.ts

```diff
@@ -171,4 +171,7 @@ export const hardforksDict: HardforksDict = {
   verkle: {
     eips: [4762, 6800],
   },
+  evmmax: {
+    eips: [6690],
```

Comment:
I don't think we should do an `evmmax` hardfork for this: if EVMMAX should be activated then the 6690 EIP should be activated (so instantiate a common and activate 6990 to use EVMMAX). I might have missed something from the EIP though. It seems to me that the EIP introduces new opcodes?